models/groups_test.go

```diff
@@ -249,3 +250,34 @@ func TestGroup_BuildSuccessfulJsonResponse(t *testing.T) {
 		So(userJson["precedence"], ShouldEqual, precedence)
 	})
 }
+
+func TestGroup_BuildListUsersInGroupRequestWithNextToken(t *testing.T) {
+	Convey("builds a correctly populated Cognito ListUsersInGroup request body without a nextToken", t, func() {
+		group := models.Group{
+			Name: "role-test",
+		}
+		userPoolId := "euwest-99-aabbcc"
+		nextToken := ""
+
+		response := group.BuildListUsersInGroupRequestWithNextToken(userPoolId, nextToken)
+
+		So(reflect.TypeOf(*response), ShouldEqual, reflect.TypeOf(cognitoidentityprovider.ListUsersInGroupInput{}))
+		So(*response.UserPoolId, ShouldEqual, userPoolId)
+		So(*response.GroupName, ShouldEqual, group.Name)
```

Comment:
Add an assertion to verify that `NextToken` is nil/empty

Reply:
I did initially try with this ```So(*response.NextToken, ShouldBeNil)``` but I got the following error: ```runtime error: invalid memory address or nil pointer dereference ```. Do you know if there is a way to write a test without getting this error please?

Reply:
I think it works if I have `So(response.NextToken, ShouldBeNil)`